src/Features/Core/Portable/Wrapping/InitializerExpression/AbstractInitializerExpressionWrapper.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Indentation;
+
+namespace Microsoft.CodeAnalysis.Wrapping.InitializerExpression
+{
+    internal abstract partial class AbstractInitializerExpressionWrapper<
+        TListSyntax,
+        TListItemSyntax>
+        : AbstractSyntaxWrapper
+        where TListSyntax : SyntaxNode
+        where TListItemSyntax : SyntaxNode
+    {
+        protected abstract string Unwrap_list { get; }
+        protected abstract string Wrap_long_list { get; }
+        protected abstract string Unwrap_all_items { get; }
+        protected abstract string Wrap_every_item { get; }
+        protected abstract string Indent_all_items { get; }
+
+        protected abstract bool DoWrapInitializerOpenBrace { get; }
```

Comment:
I do wish this got a doc comment explaining it's behavior (but see concern in AbstractCSharpInitializerExpressionWrapper.cs of whether it's correct.)

Reply:
I will have to double check, but I believe this property is still relevant because for vb we want to skip moving the open brace to the next line. I thought it would be better to pass in the parameter instead of checking the language the file is written in. For c#, I will definitely check the user's setting. I will definitely add a doc comment explaining behavior.

Reply:
I removed this code, but have a similar concept for checking what to do with the opening brace. I didn't add a doc comment, but I definitely can if the new method isn't more clear.